src/index.js

```diff
@@ -12,8 +12,40 @@ import { graphqlExpress, graphiqlExpress } from 'apollo-server-express';
 import { makeExecutableSchema } from 'graphql-tools';
 import resolvers from './graphql/resolvers';
 import typeDefs from './graphql/schema';
+import swagger from 'express-swagger-generator';
 
 let app = express();
+
+const expressSwagger = swagger(app);
```

Comment:
Maybe this should be optional - enabled by the config based property - not sure if everybody are willing to have swagger on?

Reply:
Did some refactoring. currently only enable/disable via config, for later we can load the swagger options also from the config.